synapseclient/core/credentials/cred_data.py

```diff
@@ -96,10 +96,10 @@ def __call__(self, r):
         return r
 
     def __repr__(self):
-        return  f"SynapseAuthTokenCredentials("\
-            f"username='{self.username}', "\
-            f"displayname='{self.displayname}', "\
-            f"token='{self.secret}')"
+        return (f"SynapseAuthTokenCredentials("
+            f"username='{self.username}', "
+            f"displayname='{self.displayname}', "
```

Comment:
@danlu1 / @BryanFauble Thoughts on just removing displayname?

Reply:
I'm open to either discard displayname or check displayname first otherwise use username.

Reply:
If synapse is giving us the data I am in favor of leaving it in.

Reply:
Lets find out if it is actually deprecated? Then go from there.

Reply:
Works for me. @danlu1 Do you mind following up with Platform on this item?

Reply:
Sure. I tested it with my personal account and DPE test account. With my personal account, `syn.restGET("/userProfile", auth=credentials)` gives 

```{'ownerId': '3434599', 'etag': '01f8652a-55b8-4816-b8b5-cd8d3e479feb', 'firstName': 'Dan', 'lastName': 'Lu', 'emails': ['dan.lu@sagebase.org'], 'openIds': [], 'userName': 'danlu', 'displayName': 'Dan Lu', 'summary': '', 'position': 'Research Software Engineer', 'location': 'Seattle, WA, USA', 'industry': '', 'company': 'Sage Bionetworks', 'profilePicureFileHandleId': '81598436', 'url': '', 'notificationSettings': {'sendEmailNotifications': True}, 'createdOn': '2021-09-15T16:12:59.000Z', 'twoFactorAuthEnabled': False}```

And with the DPE test account,
```
{'ownerId': '3336428', 'etag': '9b4a8f96-ffe7-41bc-b4c9-82d198f8b176', 'emails': ['test@sagebase.org'], 'openIds': [], 'userName': 'test', 'notificationSettings': {'sendEmailNotifications': True}, 'createdOn': '2021-09-11T06:53:31.000Z', 'twoFactorAuthEnabled': False}
```

It seems that the displayName will be given back if it exists for the account.

Reply:
Tracked [here](https://sagebionetworks.jira.com/browse/PLFM-8522).